scss/components/_sheet.scss

```diff
@@ -219,3 +198,15 @@ nav.sheet-tabs {
 .brute h2.ability-title {
   text-shadow: none;
 }
+
```

Comment:
Sorry for reformat... the real change is on this lines

Reply:
👍